src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+@import "./nav.scss";
+
+.header {
+  display: flex;
```

Comment:
the same here - "@use". Or you should move this import to main.scss

Reply:
nav.scss need for the header block work. If i move @import "./nav.scss"; to main.scss and then use header block in other project, nav not works.